Util_Environment.php

```diff
@@ -1208,6 +1208,34 @@ static public function is_flushable_post( $post, $module, $config ) {
 
 
 
+	/**
+	 * Checks if post is a custom post type
+	 *
+	 * @since 2.1.7
+	 * 
+	 * @param unknown $post
+	 *
+	 * @return bool
+	 */
+	static public function is_custom_post_type( $post ) {
```

Comment:
can you optimize it a bit please?

there's `get_post_type_object( $post_type )` returning object instead of list.